static/docs/user-guide/dvcignore.md

```diff
@@ -1,15 +1,15 @@
 # .dvcignore File
 
-Marks which files and/or directories should be ignored when traversing
-repository.
+Marks which files and/or directories should be excluded when traversing a
```

Comment:
why not ignored? feels like it can be a better term since it's _dvcignore_ :)

Reply:
The term "ignore" is everywhere on that document. I added a couple synonyms like "exclude" and "omit" for user search purposes. The former ("exclude") is also used in other docs in reference to `.dvcignore` usage, but mostly for other purposes. "Omit" is not used anywhere else.

If you'd like I can revise all the occurrences of these terms to make sure only "ignore" is used for Dvcignore stuff, and that "exclude" is also used consistently (in some other context). Maybe incorporate "omit" if needed. Lmk

Reply:
p.s. It's also good writing practice to have varied lexicon.

Reply:
👍